src/tenants/planet/LeaderBoard/index.tsx

```diff
@@ -14,58 +13,11 @@ export default function index({
   leaderboard,
   tenantScore,
 }: Props): ReactElement {
-  const blockedCountries = [
```

Comment:
These blockedCountries won't be required in the future as well? Were these video specific?

Reply:
This array does not seem to be used somewhere else.

Reply:
I'm not sure, but I've documented the list in `Video` as a comment.